sdcm/logcollector.py

```diff
@@ -742,6 +742,10 @@ class ScyllaLogCollector(LogCollector):
                                command='for i in /var/tmp/scylla/*.log;do echo [$i]; cat $i;done'),
                     CommandLog(name='scylla_doctor.vitals.json',
                                command='cat *.vitals.json'),
+                    CommandLog(name='cloud-init-output.log',
+                               command='cat /var/log/cloud-init-output.log'),
+                    CommandLog(name='cloud-init.log',
+                               command='cat /var/log/cloud-init.log'),
```

Comment:
There is TODO comment in another part of the code:

https://github.com/scylladb/scylla-cluster-tests/blame/c5cb0f14/sdcm/sct_provision/instances_provider.py#L55

Shouldn't it be updated in this case? -

```
    for v_m in provisioned_instances:
        ssh_login_info = {'hostname': v_m.public_ip_address,
                          'user': v_m.user_name,
                          'key_file': f"~/.ssh/{v_m.ssh_key_name}"}
        remoter = RemoteCmdRunnerBase.create_remoter(**ssh_login_info)
        wait_cloud_init_completes(remoter=remoter, instance=v_m)
        # todo: wait for scylla-machine-image service to complete if instance is scylla-db?
        # todo: download cloud-init logs
```

Reply:
@soyacz care to remove this comment ? before we merge this one ?

Reply:
removed